### PR TITLE
Add title to campaign edit

### LINKED ISSUE #840: Add the Title of the campaign above "This campaign is running!"
It would be nice to have the name of the campaign above "This campaign is running."

image

### PR DESCRIPTION
**Pull request by higgyCodes**
# Fixes #840 

## Description

As above, a simple one liner to add the campaign title to the campaign edit view. Title updates if a valid title is submitted successfully in the "Basics" section.

image

### CODE REVIEW

File: src/containers/AdminCampaignEdit.jsx [hunk 1]
```diff
@@ -502,31 +502,34 @@ class AdminCampaignEdit extends React.Component {
...
-          <div
-            style={{
-              color: theme.colors.gray,
-              fontWeight: 800
-            }}
-          >
-            <CircularProgress
-              size={0.5}
+      <div>
+        <h2>{this.props.campaignData.campaign.title}</h2>
```
Comment (schuyler1d) [inline src/containers/AdminCampaignEdit.jsx]: what happens with a new campaign?  Maybe we should condition the `h2` on a title exisitng?

**Review by schuyler1d — DISMISSED** [review]
I think this is fine as-is, but maybe better to just condition the h2 on a title being present

File: src/containers/AdminCampaignEdit.jsx (hunk 1 above)
Comment (higgyCodes) [reply src/containers/AdminCampaignEdit.jsx]: A new campaign appears to always contain the title property value 'New Campaign' as seen below:

image

 Given that the value appeared to be guaranteed by the validation, I omitted any additional conditionality. If this feels presumptuous, I'd be happy to add it in.

**Review by schuyler1d — APPROVED** [review]
Cool, thanks!

**Pull request merged**